copyright

```diff
@@ -2165,6 +2165,41 @@ Files:
 Copyright: RisingLeaf (https://github.com/RisingLeaf)
 License: CC-BY-SA-4.0
 
+Files:
+ images/scene/volcanic-eruption-at-night.jpg
+Copyright: None; public domain. Creator Vide/A.Hubner, US National Park Service (https://www.nps.gov/media/video/view.htm?id=2F152320-6D7D-4CF1-875D-37B3F3216293)
+License: None; public domain.
+
+Files:
+ images/scene/blue-whale.jpg
+Copyright: Mike Baird (https://commons.wikimedia.org/wiki/File:Blue_Whale_%28Balaenoptera_musculus%29_Mysticeti_baleen_whale.jpg, https://www.flickr.com/people/72825507@N00)
+License: Creative Commons Attribution 2.0 Generic
+
+Files:
+ images/scene/purple-lightning.jpg
+Copyright: None; public domain. Created by Sean Waugh in US NOAA NSSL (https://photolib.noaa.gov/Collections/National-Severe-Storms-Laboratory/Lightning/emodule/465/eitem/435)
+License: None; public domain.
+
+Files:
+ images/land/rough-seas.jpg
+Copyright: None; public domain. Created by Art Howard in US NOAA (https://oceanexplorer.noaa.gov/multimedia/daily-image/media/20201108.html)
+License: None; public domain.
+
+Files:
+ images/land/monterey-bay-aquarium.jpg
+Copyright: None; public domain. Created by Brianna Shaughnessy in US NOAA (https://www.noaa.gov/office-education/celc/news/giving-thanks-to-partners-noaa-education-and-aquariums-gather-for-4th-collaborative-workshop)
+License: None; public domain.
```

Comment:
```suggestion
Files:
 images/scene/purple-lightning*
 images/land/rough-seas*
 images/land/monterey-bay-aquarium*
Copyright: N/A (public domain)
License: public-domain
Comment:
 Taken from https://archive.is/VzbdB, https://archive.is/NmmVG, and https://archive.is/Tr5I7, respectively. From US NOAA, and therefore in the public domain because they were created by government employees while doing work for the government.
```